examples/pipelines/fl_cross_silo_basic/submit.py

```diff
@@ -123,15 +126,30 @@ def custom_fl_data_path(datastore_name, output_name, unique_id="${{name}}", roun
     Returns:
         data_path (str): direct url to the data path to store the data
     """
-    data_path = f"azureml://datastores/{datastore_name}/paths/federated_learning/{unique_id}/{output_name}/"
+    data_path = f"azureml://datastores/{datastore_name}/paths/federated_learning/{output_name}/{unique_id}/"
     if round:
         data_path += f"round_{round}/"
 
     return data_path
 
 
+def getUniqueIdentifier(length):
+    """Generates a random string and concatenates it with today's date
+
+    Args:
+        length (int): length of the random string
+
+    """
+    str = string.ascii_lowercase
+    date = datetime.date.today().strftime("%B_%d_%Y_")
```

Comment:
I suggest YYYY-MM-DD instead this format, as it will be easier to identify the results of a given run

Reply:
Ok. Done